### PR TITLE
Description of incremental analysis using skipfile

### PR DESCRIPTION
**Pull request by dkrupp**
Incremental analysis using skip file and gerrit integration possibility
is added to the usage guide.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -117,9 +130,35 @@ vi ./cmd-find.c
...
 
+#### Using skip file
+
+If you want to re-analyze only the changed source files without build, you
+can give a skip-list to CodeChecker.
+
+Let's assume that only `cmd-find.c` that needs to be re-analyzed.
+
+You need to create the following skip list file that tells CodeChecker
+to analyze analyze `cmd-find.c` and ignore the rest.
```
Comment (csordasmarton) [inline docs/usage.md]:
```suggestion
to analyze `cmd-find.c` and ignore the rest.
```

File: docs/usage.md [hunk 2]
```diff
@@ -434,6 +474,19 @@ CodeChecker cmd diff -b tmux_master_2017_08_28 -n tmux_master_2017_08_29 --new -
 Please find a [Shell Script](script_daily.md) that can be used
 in a Jenkins or any other CI engine to report new bugs.
 
+###Gerrit Integration <a name="gerrit"></a>
+
+The workflow basd on *Alternative 1)* can be used to implement the gerrit integration with CodeChecker.
```
Comment (csordasmarton) [inline docs/usage.md]:
```suggestion
The workflow basd on *Alternative a)* can be used to implement the gerrit integration with CodeChecker.
```

File: docs/usage.md [hunk 3]
```diff
@@ -434,6 +474,19 @@ CodeChecker cmd diff -b tmux_master_2017_08_28 -n tmux_master_2017_08_29 --new -
 Please find a [Shell Script](script_daily.md) that can be used
 in a Jenkins or any other CI engine to report new bugs.
 
+###Gerrit Integration <a name="gerrit"></a>
```
Comment (csordasmarton) [inline docs/usage.md]:
```suggestion
### Gerrit Integration <a name="gerrit"></a>
```

File: docs/usage.md (hunk 2 above)
Comment (dkrupp) [reply docs/usage.md]: No, in this case Alternative 1) is correct. I was referring to to section directly above.

**Pull request merged**